automatic/lightalloy/lightalloy.nuspec

```diff
@@ -0,0 +1,32 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>lightalloy</id>
+    <title>Light Alloy Video Player</title>
+    <version>4.8.8.20161201</version>
+    <authors>Vortex Group, LLC.</authors>
+    <owners>chocolatey</owners>
+    <summary>Video Player with built-in codecs.</summary>
+    <description>
+Complete aesthetically pleasing video player with built-in codecs.
+
+__Warning:__ The installer is not silent.
+`AutoIt` will automatically click through the installer. Be careful when doing something else while installing, you might mess up the automated click-through.
+    </description>
+    <projectUrl>http://www.light-alloy.ru/</projectUrl>
+    <tags>Media Video Player NotSilent</tags>
```

Comment:
I'm annoyed by things like this, and the fact that `chocolatey` **does not reject or warn** about this. But that's not your fault.